### ISSUE TITLE
how to train the pretrained Latte model weights?

### ISSUE THREAD

**Issue by leonardodora** [PKU-YuanGroup/Open-Sora-Plan#256]
as the vae of opensora is different from Latte, the weights from latte could be able to use directly?
Or your team train a latte model from scratch?

**Comment by LinB203** [comment]
Yes, we use it directly. It adapts very quickly and the transformation is visible in about 500 steps. This is consistent with [pixart-sigma](https://arxiv.org/abs/2403.04692).

**Comment by leonardodora** [comment]
> Yes, we use it directly. It adapts very quickly and the transformation is visible in about 500 steps. This is consistent with [pixart-sigma](https://arxiv.org/abs/2403.04692).

Thanks for your reply! But if we want to scale up the param number, what should we do first, you suggest? just train a larger new latte model?

**Comment by LinB203** [comment]
> > Yes, we use it directly. It adapts very quickly and the transformation is visible in about 500 steps. This is consistent with [pixart-sigma](https://arxiv.org/abs/2403.04692).
> 
> Thanks for your reply! But if we want to scale up the param number, what should we do first, you suggest? just train a larger new latte model?

I think a pixart-alpha should be retrained.
